src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -465,19 +465,15 @@ impl StreamingMetrics {
         )
         .unwrap();
 
-        let opts = histogram_opts!(
-            "stream_barrier_align_duration",
+        let barrier_align_duration = register_guarded_int_counter_vec_with_registry!(
+            "stream_barrier_align_duration_ns",
```

Comment:
As you changed the name, let's add a "join" into name! (I always wanted to do so)

Reply:
Besides join, union and dynamic filter also utilize the same metrics.

The type of the operator is part of the label.

Reply:
> Besides join, union and dynamic filter also utilize the same metrics.

I see. My motivation is because it's a bit confusing with `merge_barrier_align_duration`, which is for the parallelism-level `Merger`. 

Idk. Either LGTM.